frontend/apps/reader/modules/readermenu.lua

```diff
@@ -297,6 +297,7 @@ end
 function ReaderMenu:onTapShowMenu()
     self.ui:handleEvent(Event:new("ShowConfigMenu"))
     self.ui:handleEvent(Event:new("ShowReaderMenu"))
+    return true
 end
 
 function ReaderMenu:onTapCloseMenu()
```

Comment:
No. The `configdialog` relies on tap event being propagated to it so that when both reader menu and config dialog is open tap on the blank area can close both widgets. It's a little tricky to have this implicit dependency. Maybe better to do a refactoring.

```
koreader $ grep -rn TapCloseMenu frontend/
frontend/ui/widget/configdialog.lua:481:        self.ges_events.TapCloseMenu = {
frontend/ui/widget/configdialog.lua:602:function ConfigDialog:onTapCloseMenu(arg, ges_ev)
frontend/apps/reader/modules/readermenu.lua:44:                self:onTapCloseMenu()
frontend/apps/reader/modules/readermenu.lua:224:    self:onTapCloseMenu()
frontend/apps/reader/modules/readermenu.lua:302:function ReaderMenu:onTapCloseMenu()
```